core/modules/user.py

```diff
@@ -228,47 +228,46 @@ def login(self, name=None, password=None, skey="", *args, **kwargs):
 
         name = name.lower().strip()
         query = db.Query(self.userModule.viewSkel().kindName)
-        res = query.filter("name.idx >=", name).getEntry()
-
-        if res is None:
-            res = {"password": {"pwhash": "-invalid-", "salt": "-invalid-"}, "status": 0, "name": {}}
+        res = query.filter("name.idx >=", name).getEntry() or {}  # might find another user; always keep a dict
```

Comment:
```suggestion
        user_entry = query.filter("name.idx >=", name).getEntry() or {}  # might find another user; always keep a dict
```
Can we rename this. I think `user_entry` is better then call it only res.